src/components/NewMessageForm/AudioRecorder/AudioRecorder.vue

```diff
@@ -327,6 +346,18 @@ export default {
 				opacity: 1;
 			}
 		}
+		&--disabled {
```

Comment:
any chance to make this global in some way ?
how are we doing it for the other buttons like the attachment one ?

Reply:
It'll be included in the button component, which I will pick up again soon!